week1/exercise-template/test/exercises.spec.js

```diff
@@ -5,6 +5,10 @@ afterEach(async () => {
     app.close();
 });
 
+afterAll(async () => {
+    app.close();
```

Comment:
One student's linux machine was having an issue with Port 3000 being occupied after running the tests, and I wonder if the server that is run during the tests wasn't closed properly. An alternative would be to run the server the tests use on a different port, but that is still dirty if the tests leave a server still running.

Reply:
Yeah i feel like we should not fix it here. But fix the issue for the student instead